src/styles/blocks/header.scss

```
@@ -0,0 +1,58 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: $fourth-color;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  font-size: 12px;
+  font-weight: 500;
+
+  &__img {
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav__items {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+
+  list-style: none;
+
+  .list__item {
+    .item__link {
+      color: $main-color;
+      display: flex;
+      align-items: center;
+      text-decoration: none;
+      text-transform: uppercase;
+      position: relative;
+      line-height: 60px;
+      transition: 0.3s;
```

make it with transform, you already do it a lot of times)
```suggestion
      ...
```